lib/scope.js

```diff
@@ -44,6 +45,10 @@ class RestifizerScope {
     return this.checkActionName(ACTIONS.SELECT, ACTIONS.SELECT_ONE, ACTIONS.COUNT);
   }
 
+  isSelectDistinct() {
+    return this.checkActionName(ACTIONS.DISTINCT);
```

Comment:
@vedi, I just wanted to separate select and distinct, because in one of our projects the same collection post-processing lead to crashes.